### DISCUSSION TITLE
Function expression

### DISCUSSION THREAD

**Discussion by ryskin** [infinitered/ignite#2346] [Q&A] [answered]
I see components created using Function expression, is it gives any benefits? I'm googled but have not found an answer, when using Function declaration inside components it's obvious because of hoisting, but I'm sure there was some real reason to create components in the way it's now.

**Comment by frankcalise** [comment, marked as answer]
One benefit is the `displayName` is set for the component versus declaring it with an arrow function.

**Reply by ryskin** [reply]
thank you, so it's easier to debug
